src/spinner.test.visuals.ts

```diff
@@ -8,10 +8,6 @@ for (const story of stories.Spinner) {
     for (const theme of story.themes) {
       test.describe(theme, () => {
         test('size="large"', async ({ page }, test) => {
-          await page.evaluate(
-            () => matchMedia('(prefers-reduced-motion: reduce)').matches,
```

Comment:
Turns out this wasn't necessary. Playwright disables animations before taking a screenshot.

Reply:
TIL